src/Version.php

```diff
@@ -8,10 +8,14 @@
 
 class Version
 {
-    public const string SCRIPTS_DIRECTORY = "src/scripts/";
+    public const string SCRIPTS_DIRECTORY = __DIR__ . "/scripts/";
+    public const string PATH_TO_VERSION_SCRIPT = "cd ../../../ && ./version/src/scripts/version.sh";
+    public const string PATH_TO_CHECK_SCRIPT = "cd ../../../ && ./version/src/scripts/check.sh";
 
     public function __construct(
         public bool $long = false,
+        public string $pathToVersionScript = self::PATH_TO_VERSION_SCRIPT,
+        public string $pathToCheckScript = self::PATH_TO_CHECK_SCRIPT,
```

Comment:
```suggestion
        public string $pathToVersionScript = self::PATH_TO_VERSION_SCRIPT,
        public string $pathToCheckScript = self::PATH_TO_CHECK_SCRIPT,
        public bool $long = false,
```

changing the order here will allow you to use named arg for bool and positional args for others